src/progressbar/progressbar.js

```diff
@@ -54,6 +54,7 @@ angular.module('ui.bootstrap.progressbar', [])
         require: '^progress',
         scope: {
             value: '=',
+            max: '=?',
```

Comment:
the @ binding is a one way data binding - scope.max would be changeable, and would be used like `max="{{max}}"`